client/me/help/help-results/index.jsx

```diff
@@ -24,7 +39,11 @@ module.exports = React.createClass( {
 		return (
 			<div className="help-results">
 				<SectionHeader label={ this.props.header }/>
-				{ this.props.helpLinks.map( helpLink => <HelpResult key={ helpLink.link } helpLink={ helpLink } iconTypeDescription={ this.props.iconTypeDescription } /> ) }
+				{ this.props.helpLinks.map( helpLink =>
+					<HelpResult
+						key={ helpLink.link } helpLink={ helpLink } iconTypeDescription={ this.props.iconTypeDescription }
+						onClick={ this.props.onClick ? this.createOnClick( this.props.onClick, helpLink ) : undefined } /> )
```

Comment:
This is not a big deal but we'll probably end up re-rendering the `HelpResult` component needlessly because `createOnClick` will always return a different function. We could just edit the `HelpResult` component to pass along `helpLink` in the `onClick` and it would make life a bit easier here. Then you could just do `onClick={ this.props.onClick }`